ur_robot_driver/test/robot_driver.py

```diff
@@ -249,6 +249,20 @@ def test_trajectory_scaled(self, tf_prefix):
         )
         self.assertEqual(result.error_code, FollowJointTrajectory.Result.SUCCESSFUL)
 
+    def test_tool_contact(self, tf_prefix):
+        if self._io_status_controller_interface.get_version().major < 5:
+            start_result = self._io_status_controller_interface.start_tool_contact()
+            self.assertEqual(start_result.success, False)
+
+            end_result = self._io_status_controller_interface.end_tool_contact()
+            self.assertEqual(end_result.success, False)
+        else:
+            start_result = self._io_status_controller_interface.start_tool_contact()
+            self.assertEqual(start_result.success, True)
+
+            end_result = self._io_status_controller_interface.end_tool_contact()
+            self.assertEqual(end_result.success, True)
+
```

Comment:
This doesn't seem like it belongs here.

Reply:
This is a re-implementation of the tool contact test, so it will work on all versions of the robot. The one included in the [Tool contact PR](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/940) will only work on e-series robots, as it cannot see the robot's software version, and tool contact can only be used on e-series robots. 
I could move this to it's own pull request if that makes more sense? I agree that it probably doesn't quite fit in here.

Reply:
@fmauch I have removed the tool contact parts from both pull requests now. I will make a new PR to add the tests back when this has been merged.